js/mexc3.js

```diff
@@ -0,0 +1,3659 @@
+                askVolume = undefined;
+            }
+            baseVolume = this.safeString (ticker, 'volume');
+            quoteVolume = this.safeString (ticker, 'quoteVolume');
+            open = this.safeString (ticker, 'openPrice');
+            high = this.safeNumber (ticker, 'highPrice');
+            low = this.safeNumber (ticker, 'lowPrice');
+            prevClose = this.safeString (ticker, 'prevClosePrice');
+            changeValue = this.safeString (ticker, 'priceChange');
+            changePcnt = this.safeString (ticker, 'priceChangePercent');
+            changePcnt = this.parseNumber (Precise.stringMul (changePcnt, '100'));
+        }
+        return this.safeTicker ({
+            'symbol': market['symbol'],
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'open': open,
+            'high': high,
+            'low': low,
+            'close': this.safeString (ticker, 'lastPrice'),
+            'bid': bid,
+            'bidVolume': bidVolume,
+            'ask': ask,
+            'askVolume': askVolume,
+            'vwap': undefined,
+            'previousClose': prevClose,
+            'change': changeValue,
+            'percentage': changePcnt,
+            'average': undefined,
+            'baseVolume': baseVolume,
+            'quoteVolume': quoteVolume,
+            'info': ticker,
+        }, market, false);
+    }
+
+    async fetchBidsAsks (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchBidsAsks', undefined, params);
+        let method = undefined;
+        if (marketType === 'spot') {
+            method = 'spotPublicGetTickerBookTicker';
+            //
+            //     [
+            //       {
+            //         "symbol": "AEUSDT",
+            //         "bidPrice": "0.11001",
+            //         "bidQty": "115.59",
+            //         "askPrice": "0.11127",
+            //         "askQty": "215.48"
+            //       },
+            //     ]
+            //
+        } else if (marketType === 'swap') {
+            throw new BadRequest (this.id + ' fetchBidsAsks() is not available for ' + marketType + ' markets');
+        }
+        const response = await this[method] (query);
+        return this.parseTickers (response, symbols);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (market['spot']) {
+            return await this.createSpotOrder (market, type, side, amount, price, params);
+        } else if (market['swap']) {
+            return await this.createSwapOrder (market, type, side, amount, price, params);
+        }
+    }
+
+    async createSpotOrder (market, type, side, amount, price = undefined, params = {}) {
+        const symbol = market['symbol'];
```

Comment:
I can't create market orders although this seems to be following the docs spec, waiting on a MEXC clarification
![image](https://user-images.githubusercontent.com/43336371/164733741-3eb9eb65-0948-4a74-90fc-c29fac13b6ae.png)

Reply:
left for you too.

Reply:
done